.claude/skills/test-writer/references/fixture-guide.md

```
@@ -0,0 +1,53 @@
+        return "";  // 실제 S3 업로드하지 않음
+    }
+
+    @Override
+    public String issueSignedUrl(String path) {
+        return "test-prefix-" + path + "-test-suffix";
+    }
+}
+```
+
+인수 테스트에서는 `@MockitoBean`으로 Mock 처리:
+```java
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)